src/Thinktecture.Relay.Server/Services/DiscoveryDocumentBuilder.cs

```diff
@@ -45,7 +45,9 @@ public DiscoveryDocument BuildDiscoveryDocument(HttpRequest request)
 				ResponseEndpoint = new Uri(baseUri, "body/response").ToString(),
 				ConnectionTimeout = (int)TimeSpan.FromSeconds(30).TotalSeconds,
 				ReconnectMinimumDelay = _relayServerOptions.ReconnectMinimumDelay,
-				ReconnectMaximumDelay = _relayServerOptions.ReconnectMaximumDelay
+				ReconnectMaximumDelay = _relayServerOptions.ReconnectMaximumDelay,
+				HandshakeTimeout = _relayServerOptions.HandshakeTimeout,
+				KeepAliveInterval = _relayServerOptions.KeepAliveInterval
```

Comment:
Add comma, then we don't have a unnecessary change of this line when we add new values like we had this time on the `ReconnectMaximumDelay`, but only an addition at the bottom.

```suggestion
				KeepAliveInterval = _relayServerOptions.KeepAliveInterval,
```